src/main/thrift/parquet.thrift

```diff
@@ -349,6 +349,8 @@ enum CompressionCodec {
   GZIP = 2;
   LZO = 3;
   BROTLI = 4;
+  LZ4 = 5;
+  ZSTD = 6;
```

Comment:
should we add a comment to clarify what is expected to be supported?

Reply:
BROTLI LZ4 an ZSTD are new. Possibly add since v x.y.z on each to manage expectations?

Reply:
Good point. I'll add the versions they were added.